cdk/package.json

```diff
@@ -17,9 +17,10 @@
 		"diff": "cdk diff --path-metadata false --version-reporting false"
 	},
 	"devDependencies": {
+		"@guardian/eslint-config": "^5.0.0",
```

Comment:
This is redundant with `@guardian/eslint-typescript-config`, which includes the javascript one automatically. We do not need this.

Reply:
Thanks. I removed this.